addons/aircraft/functions/fnc_droneSetWaypoint.sqf

```diff
@@ -36,7 +36,30 @@ _pos set [
 
 // [_group] call CBA_fnc_clearWaypoints;
 _waypoint = _group addWaypoint [_pos, 0];
-_waypoint setWaypointType _type;
+if (_type == "FOLLOW" && {["CAManBase", "LandVehicle", "Ship"] findIf {_target isKindOf _x} != -1}) then {
+    _waypoint setWaypointType "HOLD";
+    //_waypoint waypointAttachVehicle _target;
+    [{
+        params ["_args", "_handle"];
+        _args params ["_vehicle", "_group", "_waypoint", "_target"];
+
+        if ( // Abort PFH if a new waypoint is created via UAV Terminal or ACE Interaction
+            _waypoint select 1 != currentWaypoint _group || 
+            {!alive _vehicle}
+        ) exitWith {
+            [_handle] call CBA_fnc_removePerFrameHandler;
+        };
+
+        private _followDistance = _vehicle getVariable [QGVAR(wpFollowDistance), 0];
+        if ((_vehicle distance2D _target) < _followDistance) then {
+            _waypoint setWaypointPosition [(getPosASL _vehicle), -1];
+            systemChat "waypoint updated, waiting.";
```

Comment:
I'm not sure if we want `systemChat` being spammed with this.
If we do, this would have to be a localized stringtable entry.

Reply:
My bad, that too is a leftover from testing.
Actually, I'm still in doubt about one last thing, should [air vehicles be follow-able too](https://github.com/acemod/ACE3/pull/9889#issuecomment-2066336074)?